tensorflow/python/compiler/tensorrt/test/testdata/gen_tf_readvariableop_model.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""Saves a TensorFlow model containing ReadVariableOp nodes.
+
+   The saved model is loaded and executed by tests to check that TF-TRT can
+   successfully convert and execute models with variables without freezing.
+"""
+
+import numpy as np
+import tensorflow as tf
```

Comment:
'import tensorflow' statements are no longer allowed in the internal
    TensorFlow codebase, with the exception of /examples/ and /g3doc/
    directories. This is because they prevent Blaze and TAP from caching
    anything.
    
    Please import modules directly. Also please make sure your Python
    BUILD rules depend on all the labels whose sources they import.